src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -1282,6 +1283,9 @@ namespace cryptonote
           case relay_method::local:
             private_req.txs.push_back(std::move(std::get<1>(tx)));
             break;
+          case relay_method::forward:
```

Comment:
Another reasonable big change. Transactions in the forward state get relayed over Dandelion++ stems when the timer expires.